cerebrium/cortex/advanced-functionality/config-files.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Cortex Config Files
+description: Using config files to configure Cortex deployments easily and quickly
+---
+
+Deploying your model with Cortex is a fast and simple process, however, there are a few parameters to keep track of and you may not want to type them out every time you deploy.
+To make this process easier, Cortex supports the use of config files.
+Config files are YAML files that contain all of the parameters that you would normally pass to the CLI.
+
+Using config files also makes it easier to keep track of your Cerebrium deployments, share them and use versioning tools such as git.
+
+## Using Config Files
+
+To deploy your model with a config file, you can use the `cortex deploy` command with the `--config-file` flag to specify the path to your config file.
+
+```bash
+cerebrium deploy --config-file your_config_file.yaml
+```
+
+Your config file can be named anything you want and can be placed anywhere on your local machine. However, remember to use the `cerebrium deploy` command in the same directory as your main.py as you would normally.
+
+You can alternatively use the `cerebrium init-cortex-project` command to generate your config file and setup a Cortex project in your desired directory. See [here](/cerebrium/cortex/advanced-functionality/init-cortex-project) for more info.
```

Comment:
is this not `init-cortex`? If it isn't can we change it to that

Reply:
Can do!